test/specs/components/Image/Image-test.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react'
```

Comment:
The changed tests are not part of 'Accessibility documentation', these should be a separate PR.

Reply:
this actually is a separate PR (https://github.com/stardust-ui/react/pull/74/files) - once that gets merged, the change should disappear from this PR.